src/components/forms/VolumeSnapshotsForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC, ReactNode } from "react";
+import { Input } from "@canonical/react-components";
+import InstanceConfigurationTable from "components/forms/InstanceConfigurationTable";
+import SnapshotScheduleInput from "components/SnapshotScheduleInput";
+import { getStorageConfigurationRow } from "pages/storage/forms/StorageConfigurationRow";
+import { useDocs } from "context/useDocs";
+import { FormikProps } from "formik";
+import { StorageVolumeFormValues } from "pages/storage/forms/StorageVolumeForm";
+
+interface Props {
+  formik: FormikProps<StorageVolumeFormValues>;
+  children?: ReactNode;
+}
+
+const VolumeSnapshotsForm: FC<Props> = ({ formik }) => {
```

Comment:
This seems like a non-generic component, better keep it in the storage directory?